src/nupic/ntypes/ArrayBase.hpp

```diff
@@ -126,33 +132,70 @@ namespace nupic
     SDR* getSDR();
     const SDR* getSDR() const;
 
-    // number of elements of given type in the buffer
+    /**
+     * number of elements of given type in the buffer
+     */
     size_t getCount() const;
 
-    // max number of elements this buffer can hold (capacity)
+    /**
+     * max number of elements this buffer can hold (capacity)
+     */
 	  size_t getMaxElementsCount() const;
```

Comment:
nit whitespace

Reply:
> nit whitespace

? it aligns on my screen.  I will see if there is a tab in there.  Actually I think I have tabs all converted to spaces.